mapflow/mapflow.py

```diff
@@ -2035,21 +2050,9 @@ def post_processing_error_handler(self, response: QNetworkReply) -> None:
 
         :param response: The HTTP response.
         """
-        error = response.error()
```

Comment:
response_body wasn't defined, so there was a Python error.
But if we define it here and use readAll, then, when we pass response to report_http_error, we can't read it there. 
So, I moved this part to report_http_error as well, so we can read response one time:
```
if error == QNetworkReply.ContentAccessDenied and "data provider" in response_body.lower():
        self.alert(self.tr('Data provider with id is unavailable on your plan. \n '
                           'Upgrade your subscription to get access to the data. \n'
                           'See pricing at <a href=\"https://mapflow.ai/pricing\">mapflow.ai</a>'),
                   QMessageBox.Information)
```